snc.sh

```diff
@@ -246,6 +246,13 @@ retry ${OC} replace -f pull-secret.yaml
 # Wait for the cluster again to become stable because of remove of pull secret
 wait_till_cluster_stable
 
+# Delete the pods from MCO forcefully as workaround of
+# https://bugzilla.redhat.com/show_bug.cgi?id=1965992
+retry ${OC} delete pods --all -n openshift-machine-config-operator --grace-period=0
```

Comment:
A `retry` should not be needed as we just waited for the cluster to be stable?

Reply:
@cfergeau yes it is just for a safer side, even we waited for cluster to be stable it doesn't provide any guarantee that it will no more go to reconcile  state.